Buildings/Controls/OBC/ASHRAE/G36/AHUs/SingleZone/SetPoints/ReliefDamper.mo

```diff
@@ -0,0 +1,208 @@
+﻿within Buildings.Controls.OBC.ASHRAE.G36.AHUs.SingleZone.SetPoints;
+block ReliefDamper "Control of actuated relief  dampers without fans"
+  parameter Real minRelDamPos(min=0, max=1, unit="1") = 0.1
+    "Relief damper position that maintains a building pressure of 0.05 inchWC (12.44 Pa) while the system is at MinPosMin 
```

Comment:
The figure uses lower case `minPosMin`. Please make sure that notation is consistent.